Cargo.toml

```diff
@@ -80,3 +86,11 @@ tonic = "0.12"
 [patch.crates-io]
 abscissa_core = { git = "https://github.com/str4d/abscissa.git", rev = "8da8e10cad02aa4a93099cb4942b48fbdc54eb89" }
 abscissa_tokio = { git = "https://github.com/str4d/abscissa.git", rev = "8da8e10cad02aa4a93099cb4942b48fbdc54eb89" }
+age = { git = "https://github.com/str4d/rage.git", rev = "84dc1e9f641994388f107ceebdbaa00126e30e16" }
```

Comment:
This is https://github.com/str4d/rage/pull/565.

Reply:
Reviewed and checked the hash.